scripts/generateDocs.js

```diff
@@ -278,10 +280,15 @@ async function main() {
             Nav: generateNavSearchResult(markdownFile)
         });
 
+        var redirectUrl = urlMapping[relativePath.substring(9)];
```

Comment:
```suggestion
        var redirectUrl = urlMapping[relativePath.substring(9)];
        if (!redirectUrl) { throw "No mapping for " + relativePath; }
```

Reply:
My thought with this change is that we should be redirecting _every_ page -- and that's why it should throw if no mapping is found.

What pages aren't being redirected?

Reply:
The only docs page that I am aware of that does not have a mapping is the "execute_process" page since there is no corresponding page on the learn site.

Reply:
As it stands, that adds a nasty message to the page:
```
If you are not redirected automatically, follow this .
```
I really think we _must_ have a redirect for every page, even if that means redirecting to the root of the new documentation.